### PR TITLE
Add issue template to pre fill some required github issue text.

### PR DESCRIPTION
**Pull request by TeoTwawki**
This was actually the file that does what I was saying in last nights pull request comment, my bad. The other one I did last night creates the link both on issues and pull requests to let you give users more verbose information about things they should or should not be doing.

### CODE REVIEW

**Comment by TeoTwawki** [comment]
I should probably not have contributing.md and style.md as separate files after all. Will see what you think after I have the examples written.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+**_Client Version_ :**
```
Comment (bendangelo) [inline ISSUE_TEMPLATE.md]: Would people really know their client version?

**Comment by bendangelo** [comment]
I think steps to reproduce and expected behaviour should be added.

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (TeoTwawki) [inline ISSUE_TEMPLATE.md]: `/ver` says hi

Comment (teschnei) [inline ISSUE_TEMPLATE.md]: They better learn if they don't

**Comment by TeoTwawki** [comment]
Making an amend to reword and clarify a few things, changed my mind about waiting - may as well make it clean now

**Comment by TeoTwawki** [comment]
Amended. [This is what it looks like now](https://github.com/TeoTwawki/darkstar/blob/master/ISSUE_TEMPLATE.md).

**Pull request merged**